packages/flutter_tools/lib/src/version.dart

```diff
@@ -254,6 +257,8 @@ class FlutterVersion {
     try {
       final String branch = _channel == 'alpha' ? 'alpha' : 'master';
       final DateTime remoteFrameworkCommitDate = DateTime.parse(await FlutterVersion.fetchRemoteFrameworkCommitDate(branch));
+      // fire and forget since nothing subsequent depends on it. Hope there's a flush/sync mechanism.
+      // ignore: unawaited_futures
```

Comment:
May be a bug. Doesn't seem to actually work on multi-line expressions when it's like:
```
aFutureReturningFunction(
  a,
  b
); // ignore: unawaited_futures
```